ledger/participant-integration-api/src/main/resources/db/migration/h2database-appendonly/V1__Append_only_schema.sql

```diff
@@ -81,6 +81,7 @@ CREATE TABLE party_entries (
     typ VARCHAR NOT NULL,
     rejection_reason VARCHAR,
     is_local BOOLEAN,
+    party_id integer NOT NULL DEFAULT 0,
```

Comment:
Actually, looking at the code, it does not seem to make use of this field does it? Could this be the wrong commit for this change?

Reply:
yes. 
* this will be filled/populated later, after data migration of interning is done
* it will be used by a pruning query
We can do this later in task INTERNING-WRITE4, but I that one is quite heavy already, that's why I was adding it here.
Have nothing against doing this later.

Reply:
Right, let's not add this upfront if it's not needed at the moment.